404-Room-Not-Found/r-squared-app
# Anthony bugfix

Branch --> Master - room show reservation bug fix, fixed issue with url bashing

--Changes--
1. added bookings to the home controller index so it can access the correct bookings
2. added validation guards to make sure you can not access actions through the path w/o login
3. some clean up on seed and the db migration files

--Testing--
1. ensure that you can log in and see your reservations
2. add a reservation through the booking dlg, make sure that it is a valid room (error checking no in place yet)
3. ensure you can see the new booking
4. log out and try to access /home or /rooms from the url, it should block you
